content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -247,20 +262,85 @@ zones, or regions.
     caption="A service mesh after a control plane instance fails"
     >}}
 
-The following list ranks control plane deployment examples by availability:
+### Independent Control Planes
+
+You can also deploy a control plane per cluster, with a shared root CA. Each cluster's control plane
+manages only the workloads in its own cluster. If the control plane is unavailable (due to upgrade or
+temporary outage), the workloads will continue to function with the existing sidecar configuration and
+will re-attempt to connect to the local control plane periodically.
+
+This model affords the following benefits:
+
+- Highest level of availability: If a control plane becomes unavailable, the scope of
+  the outage is limited to only that cluster and the workloads within it.
+
+- Maximum configuration isolation: You can make configuration changes in one cluster without impacting
+  workloads deployed in any other cluster.
+
+The following list ranks control plane deployment examples by availability, configuration isolation:
+
+- One control plane per region (**lowest availability**, **lowest isolation**)
+- Multiple control planes per region
+- One control plane per zone
+- Multiple control planes per zone
+- One control plane per cluster (**highest availability**, **highest isolation**)
+
+## Mesh models
```

Comment:
Can we move Mesh models back to where it was? It's built on top of `trust models`, so I'd like to see that come first.

Reply:
yes but trust models is also talking about multi-mesh.. So its chicken and egg.

Reply:
Then can we move it back for diff purposes at least? We can reorganize in a follow-up if we think it's necessary.